isthmus/src/main/java/io/substrait/isthmus/SqlConverterBase.java

```diff
@@ -1,11 +1,11 @@
 package io.substrait.isthmus;
 
+import com.github.bsideup.jabel.Desugar;
 import io.substrait.extension.SimpleExtension;
 import io.substrait.isthmus.calcite.SubstraitOperatorTable;
 import io.substrait.type.NamedStruct;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
```

Comment:
@davisusanibar I think it is better to minimize `.*` type imports.

Reply:
Changed